### PR TITLE
Improve Q&A's `Does KeePassXC work on mobile phones?`

### PR DESCRIPTION
**Pull request by sarpik**
There are probably technical incorrections I've introduced - feel free to suggest changes
(@PhilippC you're definitely qualified for this :D )

Also, there're probably some sentences you'd like to change - feel free:)

Fixes https://github.com/keepassxreboot/keepassxreboot.github.io/issues/61, improves https://github.com/keepassxreboot/keepassxc/issues/1060

This is how the q&a looks like after the change:

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -469,18 +469,25 @@ <h5>Platform-specific</h5>
...
         <dd>
-            No, KeePassXC only works on desktop systems. Porting it properly to mobile platforms would require a full rewrite.
-            You may be able to compile it for the mobile OS of your choice, but KeePassXC isn't at all optimized
+						We don't have our own mobile app, but you can have the same functionality on both Android and iOS!
+						<br>
+						For Android, we recommend <a href="https://play.google.com/store/apps/details?id=keepass2android.keepass2android">KeePass2Android</a>.
+						<br>
+						(it's <a href="https://github.com/PhilippC/keepass2android">open-sourced on GitHub</a>,
+						<a href="https://github.com/keepassxreboot/keepassxc/issues/1060#issuecomment-442347384">works for kdbx4 databases</a>
+						and even supports YubiKeys via NFC!)
```
Comment (droidmonkey) [inline docs/index.html]: You can remove most of this information. Just mention the fact that it is open source and provide the Github link. No need for the exclamation points either.

File: docs/index.html [hunk 2]
```diff
@@ -469,18 +469,25 @@ <h5>Platform-specific</h5>
...
+						<br>
+						And for iOS, we suggest <a href="https://itunes.apple.com/us/app/strongbox-password-safe/id897283731">Strongbox</a>
+						(also <a href="https://github.com/mmcguill/StrongBox">open-sourced on GitHub</a>!)
```
Comment (droidmonkey) [inline docs/index.html]: Likewise

File: docs/index.html [hunk 3]
```diff
@@ -469,18 +469,25 @@ <h5>Platform-specific</h5>
...
+						For Android, we recommend <a href="https://play.google.com/store/apps/details?id=keepass2android.keepass2android">KeePass2Android</a>.
+						<br>
+						(it's <a href="https://github.com/PhilippC/keepass2android">open-sourced on GitHub</a>,
+						<a href="https://github.com/keepassxreboot/keepassxc/issues/1060#issuecomment-442347384">works for kdbx4 databases</a>
+						and even supports YubiKeys via NFC!)
+						<br>
+						And for iOS, we suggest <a href="https://itunes.apple.com/us/app/strongbox-password-safe/id897283731">Strongbox</a>
+						(also <a href="https://github.com/mmcguill/StrongBox">open-sourced on GitHub</a>!) 
+						<br>
+            For KeePassXC, porting it properly to mobile platforms would require a full rewrite.
```
Comment (droidmonkey) [inline docs/index.html]: Remove this paragraph entirely.

File: docs/index.html [hunk 4]
```diff
@@ -469,18 +469,25 @@ <h5>Platform-specific</h5>
...
+            You may be able to compile KeePassXC for the mobile OS of your choice, but it isn't at all optimized
             for mobile screen sizes and form factors, let alone multi-touch input. We also don't see any advantage in
-            providing a mobile version of KeePassXC when there are excellent alternatives, already. For Android we
-            recommend <a href="https://play.google.com/store/apps/details?id=keepass2android.keepass2android">KeePass2Android</a>
-            and for iOS <a href="https://itunes.apple.com/us/app/strongbox-password-safe/id897283731">Strongbox</a>. Other
-            alternatives are <a href="https://itunes.apple.com/us/app/minikeepass/id451661808?mt=8">MiniKeePass</a>
+						providing a mobile version of KeePassXC when there are excellent alternatives (as mentioned above), already. 
+						<br>
+            Other alternatives are <a href="https://itunes.apple.com/us/app/minikeepass/id451661808?mt=8">MiniKeePass</a>
             (appears abandoned, no KDBX4 support as of now) and <a href="https://itunes.apple.com/us/app/keepass-touch/id966759076?mt=8">KeePass
```
Comment (droidmonkey) [inline docs/index.html]: I wouldn't even mention minikeepass, it is abandoned.

File: docs/index.html (hunk 3 above)
Comment (sarpik) [reply docs/index.html]: So iOS is not supported then, huh?
Or is it just the `For KeePassXC, porting <...>` part?

File: docs/index.html (hunk 4 above)
Comment (sarpik) [reply docs/index.html]: Should we mention `KeePass Touch` or just remove the `Other alternatives` section?

File: docs/index.html (hunk 3 above)
Comment (droidmonkey) [reply docs/index.html]: The porting part

File: docs/index.html (hunk 4 above)
Comment (droidmonkey) [reply docs/index.html]: Just remove it. Strongbox seems to be the only viable alternative at this time.

**Comment by sarpik** [comment]
I'll soon make the changes, currently out of time.

**Comment by sarpik** [comment]
Alright, here we are - code review welcome:)

**Comment by droidmonkey** [comment]
Great job!

**Comment by sarpik** [comment]
Thanks! I'm super happy with how keepassxc turned out with archlinux, i3 & my android phone, and even find out about syncthing to sync the database between them. Thanks a lot!

**Pull request merged**